Dockerfile

```diff
@@ -8,8 +8,12 @@ RUN go get -tags 'postgres' -u github.com/golang-migrate/migrate/cmd/migrate
 ENV GO111MODULE on
 
 RUN go get -d -v ./...
-RUN go install -v ./...
+RUN go build -v ./...
+
+FROM ubuntu:bionic
```

Comment:
my base image of choice; its about 70mb and very usable for debugging production containers.